test/functionalTest/harnessFunctions.sh

```diff
@@ -289,9 +290,10 @@ function brokerStartAwait
 
   sleep .5
 
-  # Check that CB started fine
-  curl -s localhost:${port}/version | grep version > /dev/null
+  # Check that CB started
+  curl -s localhost:${port}/version | grep version >> /tmp/brokerStart
```

Comment:
I'd suggest to rename the file to something more meanifull taking into accont its content, e.g.

/tmp/brokerStart.counter